operations/spire/Restore_Spire_Postgres_without_a_Backup.md

```diff
@@ -50,40 +47,38 @@ The CSM release tarball is required as it contains the Spire helm chart.
 3. Get the current cached sysmgmt manifest.
 
    ```bash
-   ncn# kubectl get cm -n loftsman loftsman-sysmgmt  -o jsonpath='{.data.manifest\.yaml}' -o sysmgmt.yaml
+   ncn# kubectl get cm -n loftsman loftsman-sysmgmt  -o jsonpath='{.data.manifest\.yaml}' > spire.yaml
+
    ```
 
-4. Edit the `sysmgmt.yaml` spec.charts section to only include the `spire` chart and all its current data. (The resources specified above will be updated in the next step and the version may differ, because this is an example).
+4. Run the following command to remove non-spire charts from the spire.yaml file. This will also change the metadata.name so that it doesn't overwrite the sysmgmt.yaml file that is stored in the loftsman namespace.
```

Comment:
```suggestion
4. Run the following command to remove non-Spire charts from the `spire.yaml` file.

   This will also change the `metadata.name` so that it does not overwrite the `sysmgmt.yaml` file that is stored in the `loftsman` namespace.
```